bucoffea/monojet/definitions.py

```diff
@@ -819,7 +819,12 @@ def candidate_weights(weights, df, evaluator, muons, electrons, photons, cfg):
         ele_reco_sf = evaluator['ele_reco'](electrons.etasc, electrons.pt).prod()
     weights.add("ele_reco", ele_reco_sf)
     # ID/iso SF is not split
-    weights.add("ele_id_tight", evaluator['ele_id_tight'](electrons[df['is_tight_electron']].etasc, electrons[df['is_tight_electron']].pt).prod())
```

Comment:
One comment I previously missed: Can we put in a config switch to change between the old and new behavior?

```python
if cfg.SF.AVERAGE_DIELE_ID_SF:
	# New approach
else:
	# Old approach
```

The reason I'm asking is that I would like to be able to control this behavior independently for VBF and monojet. For VBF< we have already synced tightly with our collaborators, so we should only switch stuff like this when everyone agrees. FOr now, I'd switch only for monojet.